sdk/python/tests/data/column_mapping_test_entity.csv

```diff
@@ -0,0 +1,2 @@
+id,event_timestamp
+1001,2020-09-02T00:00:00.000
```

Comment:
Can we please add a few more rows, like 5-10. One row is asking for trouble.

Reply:
Even better to generate thousands to test how multiple partitions can affect result

Reply:
Yes, using code to generate data isn't a bad idea, as long as we avoid too much `rand()`